suite-common/wallet-types/src/backend.ts

```diff
@@ -10,10 +10,8 @@ export type BlockbookUrl = {
     tor?: boolean; // Added by TOR
 };
 
-export type BackendType = 'blockbook' | 'electrum' | 'ripple' | 'blockfrost' | 'solana';
-
 export type CustomBackend = {
-    coin: (typeof networksCompatibility)[number]['symbol'];
```

Comment:
The changes in this file are because of TS error which was not clear to me, but this way is better anyway.